validator/src/main.rs

```diff
@@ -2119,6 +2126,15 @@ pub fn main() {
                     .default_value("10")
                     .help("Minimum time that the validator should not be leader before restarting")
             )
+            .arg(
+                Arg::with_name("max_delinquent_stake")
+                    .long("max-delinquent-stake")
+                    .takes_value(true)
+                    .validator(solana_clap_utils:: input_validators::is_valid_percentage)
+                    .default_value("5")
+                    .value_name("PERCENT")
+                    .help("The maximum delinquent stake % permitted for an exit [default: 5]")
```

Comment:
You don't need the " [default: 5]" here, clap automatically adds it when the `default_value()` arg is present

Reply:
yeah missed that, adjusted now. Should I worry about the CI failures? looks like an RPC test timeout